lib/supavisor.ex

```diff
@@ -104,37 +111,63 @@ defmodule Supavisor do
     end)
   end
 
+  def terminate_global(tenant) do
+    [node() | Node.list()]
+    |> Task.async_stream(
+      fn node ->
+        %{node => :rpc.call(node, Supavisor, :dirty_terminate, [tenant], 60_000)}
+      end,
+      timeout: :infinity
+    )
+    |> Enum.reduce([], fn resp, acc ->
+      [inspect(resp) | acc]
+    end)
```

Comment:
This is completely fine, but you may also consider using https://www.erlang.org/doc/man/erpc#multicall-4 - which will separate the separate calls without spawning new tasks. Also note that `Task.async_stream` has a concurrency limit, which I don't think you care here?

Reply:
good call 👍

Reply:
updated https://github.com/supabase/supavisor/pull/160/commits/a6614b363ba4c481d329cd54d32a70cca0596072